tools/scylla-sstable.cc

```diff
@@ -3209,6 +3216,8 @@ Validate the specified sstables:
             }).get();
             dbcfg.setup_directories();
             sst_log.debug("Successfully read scylla.yaml from {} location of {}", scylla_yaml_path_source, scylla_yaml_path);
+            read_object_storage_config(dbcfg).get();
+            sst_log.debug("Successfully object storage settings");
         } else {
             dbcfg.experimental_features.set(db::experimental_features_t::all());
             sst_log.debug("Failed to read scylla.yaml from {} location of {}, some functionality may be unavailable", scylla_yaml_path_source, scylla_yaml_path);
```

Comment:
We need to deal with user providing s3 paths but failing to lookup config.
In that case, we need more than a debug log, we need to tell the user that we cannot do the operation without `scylla.yaml`.

Reply:
if the endpoint config is missing the `read_object_storage_config` will throw. Also mentioned in the documentation that once using S3 stored sstables the `--scylla-yaml-file` must be used.